### ISSUE TITLE
Error when submitting ACVR

### ISSUE THREAD

**Issue by ranweiler** [FreeAndFair/ColoradoRLA#501]
This is in the `db-performance` branch.

**Repro:**
1. County admin uploads `arapahoe-manifest.csv` and `arapahoe-regent-3-clear-CVR_Export.csv`, establishes audit board.
1. State admin sets risk limit of 50%, selects the contest with ID 322 ("Regent of the University of Colorado - At Large") to audit for reason "Close contest", seed of `12345123451234512345`, launches audit.
1. County admin clicks "Start audit", selects "Clear Winner" as the only mark for the selected contest. Click review, see the mark on the Review page as expected. Click "Submit & Next Ballot".

**Result:**
500 Server error, response from server is the plain text `runtime error, no status recorded by endpoint`.

**Payload sent to `/upload-audit-cvr`:**
```json
{
  "audit_cvr": {
    "ballot_type": "17",
    "batch_id": 3200,
    "contest_info": [
      {
        "choices": [
          "Clear Winner"
        ],
        "consensus": "YES",
        "contest": "322"
      }
    ],
    "county_id": 1,
    "id": 448,
    "imprinted_id": "9-3200-36",
    "record_id": 36,
    "record_type": "UPLOADED",
    "scanner_id": 9,
    "timestamp": "2017-08-27T19:40:43.900Z"
  },
  "cvr_id": 448
}
```

**Server log:**
```
12:40:43,905  INFO corla:379 - endpoint /upload-audit-cvr hit by localhost:8888
12:40:43,908  INFO corla:89 - Audit CVR for CVR id 448 parsed and stored as id 798
12:40:43,909 ERROR corla:130 - could not query database for CVRs
12:40:43,910 ERROR GeneralError:54 - 
java.lang.IllegalStateException: no running transaction
	at us.freeandfair.corla.persistence.Persistence.checkForRunningTransaction(Persistence.java:699)
	at us.freeandfair.corla.persistence.Persistence.getByID(Persistence.java:550)
	at us.freeandfair.corla.endpoint.ACVRUpload.endpoint(ACVRUpload.java:97)
	at us.freeandfair.corla.Main.lambda$activateEndpoints$5(Main.java:251)
	at spark.RouteImpl$1.handle(RouteImpl.java:72)
	at spark.http.matching.Routes.execute(Routes.java:61)
	at spark.http.matching.MatcherFilter.doFilter(MatcherFilter.java:130)
	at spark.embeddedserver.jetty.JettyHandler.doHandle(JettyHandler.java:50)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1568)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:564)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:317)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:128)
	at org.eclipse.jetty.util.thread.Invocable$InvocableExecutor.invoke(Invocable.java:222)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:294)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:199)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by dmzimmerman** [comment]
Is this only reproducible with this particular ACVR? Or with others?

I find this confusing as I have run many, many benchmarks and tests on this branch and never seen a submission error. I wonder if there's something malformed about the ACVR (looking closely now).

**Comment by ranweiler** [comment]
It might be reproducible with other data, I haven't tried. I just wanted to document one known bad case for ease of debugging.

**Comment by ranweiler** [comment]
Closed in #500.
